site/en/tutorials/images/transfer_learning.ipynb

```diff
@@ -69,7 +69,7 @@
         "id": "hRTa3Ee15WsJ"
```

Comment:
<p><code>ds =image_dataset_from_directory(..., shuffle=true)</code>  followed by <code>take/skip</code> doesn't seem right.</p><p> </p><p>I think you need the <code>validation_split</code> argument if you're doing that.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/docs/pull/2081/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
There are 2 directories: 1 for training and 1 for validation. Then we take 20% of the validation set and use it for testing.

The `validation_split=0.2, subset=...` arg in `tf.keras.utils.image_dataset_from_directory` only reserves data for training or validation sets, which we already have.

```
`subset` must be either "training" or "validation"
```